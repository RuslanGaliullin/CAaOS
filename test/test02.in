7
67895 86364 4507 2853 50875 78043 609157
