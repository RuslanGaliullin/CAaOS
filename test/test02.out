Built array B:
86364 2853 78043 67895 4507 50875 609157 

Calculation time = 1.2e-05
